h2o-r/tests/testdir_algos/coxph/runit_coxph_concordance.R

```diff
@@ -24,7 +24,7 @@ test.CoxPH.concordance <- function() {
     rConcordance <- unname(summary(rModel)$concordance)[1]
     hexConcordance <- h2o.performance(hexModel, data=tstdataHex)@metrics$concordance
     
-    expect_equal(rConcordance, hexConcordance)
+    expect_equal(rConcordance, hexConcordance, tolerance = 1e-3, scale = 1)
```

Comment:
Isn't the tolerance quite high? This is not complex math

Reply:
It was used in all the coxph tests so far I have seen.